### ISSUE TITLE
PowerShell recipe giving CodeVerification error

### ISSUE THREAD

**Issue by jelockwood** [autopkg/vmiller-recipes#32]
@vmiller 
As of last night your PowerShell download recipe is now giving the following error.

> WARNING: com.github.vmiller.download.PowerShell is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
> 
> Code signature verification failed. Note that all verification can be disabled by setting the variable DISABLE_CODE_SIGNATURE_VERIFICATION to a non-empty value.

I initially thought that this might have been due to you updating it and my override recipe becoming outdated and needing to update my trust. However I checked and I can see you have not changed your recipes.

I think possibly that Microsoft have made a change and your download recipe's code signature verification as below may no longer match Microsoft and need updating, although strangely it does not look like the installer or app is actually signed at all.

```
			<key>Arguments</key>
			<dict>
				<key>input_path</key>
				<string>%pathname%</string>
				<key>expected_authority_names</key>
				<array>
					<string>Developer ID Installer: Microsoft Corporation (UBF8T346G9)</string>
					<string>Developer ID Certification Authority</string>
					<string>Apple Root CA</string>
				</array>
			</dict>
			<key>Processor</key>
			<string>CodeSignatureVerifier</string>
```

I can say your regex and URL are still working.

As per the error message, when I added the following to my override recipe it has worked around the issue successfully.

```
		<key>DISABLE_CODE_SIGNATURE_VERIFICATION</key>
		<true/>
```

**Comment by vmiller** [comment]
Thank you for opening an issue.  I noticed this on my daily run as well,
but was out of the office.  I should be able to take a look at this today
or tomorrow.

**Comment by jelockwood** [comment]
@vmiller 

Additional info, it looks like last night Microsoft released both 7.2.21 and 7.4.3. I have two recipe overrides, one for arm64 and one for x64. The ARM one has downloaded 7.4.3 but the Intel one has ignored that and only download 7.2.21. With the additional key to disable code verification both run with no errors. I have looked at the Github releases and both x64 and arm64 pkgs are listed.

Can you also look at this?

**Comment by vmiller** [comment]
Code signature verification removed with https://github.com/autopkg/vmiller-recipes/pull/33

Both x64 and arm64 correctly download 7.4.3 now for me.  

@jelockwood  check your override for your Intel override to make sure MAJOR_VERSION is set to 7.4

**Comment by jelockwood** [comment]
@vmiller 

Ah! Yes, I had created a brand new override for the arm64 version during my diagnosing the original issue and hence it had included automatically the 7.4 major version but had not replaced my x64 override. I have edited that and it has now downloaded the new 7.4.3 release.

Many thanks for the quick fix.
